internal/processors/signalconvert/signalconvert.go

```diff
@@ -127,6 +127,10 @@ func pruneSignals(signals []vss.Signal) ([]vss.Signal, error) {
 			}
 		}
 
+		if signalMaxValue(*signal) {
```

Comment:
I worry that some genuine signals will get caught up in this dragnet. 255 kPa is a perfectly reasonable tire pressure, for example.